plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,790 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright John Berninger (@jberning) <john.berninger at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_backup
+short_description: Get, delete, create or update Proxmox VE backup jobs.
+version_added: 8.2.0
+description:
+  - Allows you to perform some supported operations on a backup job in a Proxmox VE cluster.
+author: Dylan Leverrier (@zerchevack)
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: full
+  action_group:
+    version_added: 9.0.0
+options:
+  state:
+    description:
+      - Set to present to create or update job.
+      - Set to absent to delete job.
+    choices: ['present', 'absent']
+    type: str
+    required: true
+  all:
+    description:
+      - Backup all known guest systems on this host.
+      - Can not be use with vmid and pool in same job
```

Comment:
```suggestion
      - Can not be use with O(vmid) and O(pool) in same job
```
Please take a look at https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation and apply it to other documentation as well.

Reply:
Fix in 923fbd76014272b458eba79cdc257d9910718e23